README.md

```diff
@@ -89,31 +89,84 @@ builds like `gcc` generate a *lot* of output), and builds timing out
 
 ## Testing conda builds locally
 
-Given a fairly empty *disposable* Ubuntu x86-64 test environment (eg,
-created with Docker, or Vagrant), it is possible to simulate *part* of
-what Travis CI will do to test building individual toolchain architectures
+Given a fairly empty *disposable* Ubuntu x86-64 test environment (e.g.,
+created with Docker or Vagrant), it is possible to simulate *part* of
+what the CI does to test building individual toolchain architectures
 locally.
 
-This can be done with something like:
+The commands from the following subsections are enough to build a package
+in a "clean" docker container based on the *ubuntu* Docker image.
 
-```
-sudo apt-get update
-sudo apt-get install wget git
+### Prerequisites
 
-# Packages from ~/.travis.yml; realpath is in coreutils in Ubuntu 18.04
-# Plus libtool and pkg-config, which are needed for openocd
-#
-# sudo apt-get install realpath
-sudo apt-get install coreutils
+The only required prerequisites are:
+* [Git](https://git-scm.com/),
+* Conda installed and initialized, e.g., using
+[Miniconda](https://docs.conda.io/en/latest/miniconda.html)
+(it automatically makes required *python3* with *pip* and *setuptools*
+available),
+* [conda-build-prepare](https://github.com/litex-hub/conda-build-prepare),
+* this repository cloned to a local directory.
+
+All of these requirements can be satisfied using the following commands:
+
+```bash
+# Install git and wget (might require using `sudo`)
+apt-get update
+apt-get install -y git wget
 
+# Download Miniconda and install in CONDA_PATH
+CONDA_PATH=~/conda
+wget -c https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
+chmod a+x Miniconda3-latest-Linux-x86_64.sh
+./Miniconda3-latest-Linux-x86_64.sh -p $CONDA_PATH -b -f
+
+# Initialize Conda in the shell
+eval "$("$CONDA_PATH/bin/conda" "shell.bash" 'hook' 2> /dev/null)"
+
+# Install `conda-build-prepare`
+python3 -m pip install git+https://github.com/litex-hub/conda-build-prepare@v0.1#egg=conda-build-prepare
+
+# Clone the litex-conda-compilers repository
 git clone https://github.com/litex-hub/litex-conda-compilers.git
 cd litex-conda-compilers
-./.travis/conda-get.sh
+```
 
-# Adapted from .travis/common.sh
-get_built_package() {
-   ./.travis/conda-env.sh render --output "$@" 2>/dev/null | grep conda-bld | grep tar.bz2 | tail -n 1 | sed -e's/-[0-9]\+\.tar/*.tar/' -e's/-git//'
-}
+### Required environment variables
+
+Most importantly, the `RECIPE` variable must be set with the appropriate
```

Comment:
This isn't quite true. The `RECIPE` argument is given to conda-build-prepare command as an argument?

Reply:
You're definitely right. I've redesigned this subsection to separate variables used in the script from information about the CBP arguments.